<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpiritHouse</class>
 <widget class="QMainWindow" name="SpiritHouse">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Press Start 2P</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pixel Spirit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/UI/icon.png</normaloff>
    <normalon>../assets/UI/icon.png</normalon>../assets/UI/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/UI/resources.qrc">:/background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="TopBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/UI/resources.qrc">:/topBar.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="GoHatchingButton">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>640</y>
      <width>422</width>
      <height>62</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/SpiritHouse/go-hatching-button.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>484</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SpiritUniverseButton">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>40</y>
      <width>482</width>
      <height>72</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/SpiritHouse/spirit-universe-button.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>484</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="SpiritsArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>902</width>
      <height>482</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="SpiritsHouseContent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>900</width>
       <height>480</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/UI/resources.qrc"/>
 </resources>
 <connections/>
</ui>
